app/assets/stylesheets/mobile/modal.scss

```diff
@@ -1,9 +1,50 @@
 // base styles for every modal popup used in Discourse
 
+@keyframes slideUp {
+  from {
+    padding-top: calc(var(--composer-vh, var(--1dvh)) * 100);
+  }
+  to {
+    padding-top: 0;
+  }
```

Comment:
As mentioned above, best if we can animate non-reflow/repaint attributes like transforms.

Reply:
Yes that was my first version but I had issues combining this + the touch delta animation